tests/unit_tests/test_wave_06.py

```diff
@@ -143,7 +147,12 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result is True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_c not in tai.inventory
+    assert item_f not in jesse.inventory
```

Comment:
Fantastic! If we wanted to be SUPER sure, we could also check that the right three items are in Tai and Jesse's inventory respectively, post-swap.